common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/AbstractFetchShuffleBlocks.java

```diff
@@ -33,21 +33,25 @@
   public final String appId;
   public final String execId;
   public final int shuffleId;
+  public final int shuffleSequenceId;
 
   protected AbstractFetchShuffleBlocks(
       String appId,
       String execId,
-      int shuffleId) {
+      int shuffleId,
+      int shuffleSequenceId) {
```

Comment:
This is backward incompatible change. It changes the `FetchShuffleBlocks` message